core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -350,3 +350,9 @@ workflows:
         name: "4.12"
         namespace: ocp
         tag: upi-installer
+  openshift-e2e-nutanix:
+    platform: nutanix
+  openshift-e2e-nutanix-ovn:
+    platform: nutanix
+  openshift-e2e-nutanix-sdn:
+    platform: nutanix
```

Comment:
Are you planning on overriding environment variables when launching these clusters?
If not, then you don't need to add these workflows.  You'll simply be able to launch your clusters via the traditional `launch` command, like:
1. `launch 4.15 nutanix`
2. `launch 4.15 nutanix,ovn`
3. `launch 4.15 nutanix,ovn-hybrid`
4. `launch 4.15 nutanix,techpreview`
5. `launch 4.15 nutanix,sdn`
6. `launch 4.15 nutanix,dualstack`

Reply:
good point, i'll remove that

Reply:
done